lib/config/fileconf.go

```diff
@@ -281,12 +281,12 @@ func MakeSampleFileConfig() (fc *FileConfig, err error) {
 	s.Commands = []CommandLabel{
 		{
 			Name:    "hostname",
-			Command: []string{"/usr/bin/hostname"},
+			Command: []string{"/bin/hostname"},
 			Period:  time.Minute,
 		},
 		{
 			Name:    "arch",
-			Command: []string{"/usr/bin/uname", "-p"},
+			Command: []string{"/bin/uname", "-p"},
```

Comment:
Since these are called via `exec.Command`, should we even use absolute paths?
System tools like `hostname` and `uname` are very likely to be in `$PATH`

Reply:
I'd prefer to remove absolute paths as well. I have a similar dynamic label in my default Teleport config and it was always wrong when I switched between macOS and Linux.

Reply:
Absolute paths have been removed.